### ISSUE TITLE
ImageData constructor() - settings parameter

### ISSUE THREAD

**Issue by nchevobbe** [mdn/browser-compat-data#22810]
### What type of issue is this?

Incomplete support data

### What information was incorrect, unhelpful, or incomplete?

The `ImageData` constructor page documents the `settings` parameter: https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData#settings 
This parameter isn't supported on Firefox (pending on https://bugzilla.mozilla.org/show_bug.cgi?id=1841326), but this isn't reflected in the page.

### What browsers does this problem apply to, if applicable?

Firefox

### What did you expect to see?

It would be nice to have more detailed data that would indicate browser supports for this specific `settings` parameter

### Did you test this? If so, how?

Yes, evaluating the snippet from https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData#imagedata_using_the_display-p3_color_space (`let imageData = new ImageData(200, 100, { colorSpace: "display-p3" });`) in Firefox DevTools console does throw

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ImageData/ImageData

### MDN metadata

I think this comes from https://github.com/mdn/browser-compat-data/blob/385f57fcd50679922df46289152a7fd31b0bae60/api/ImageData.json#L47-L86

**Comment by Elchi3** [comment]
https://github.com/openwebdocs/mdn-bcd-collector/pull/1798 added a test to the collector but no compat data has been submitted to BCD.

**Comment by caugner** [comment]
Meanwhile resolved via https://github.com/mdn/browser-compat-data/pull/26088.
